File: Java_Programs/While9.java
//wap to accept a no and print from 1 to the number entered by the user.
import java.util.*;
class While9
{
	public static void main(String []args)
	{
		int Num,i=1;

		Scanner s=new Scanner(System.in);

		System.out.println("Enter the number: ");
		Num = s.nextInt();

		while(i<=Num)
		{
			System.out.println(+ i);
			i++;
		}
	}
}

File: C_Program/while9.c
//wap to accept a no. and print from 1 to the no.entered by the user.

#include<stdio.h>
int main()
{
	int num,i=1;
	printf("\n enter the no.: ");
	scanf("%d",&num);
	while(i<=num)
	{
		printf("\n num = %d",i);
		i=i+1;
	}
}